sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -542,7 +542,9 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase {
   }
 
   def operationOnlySupportedWithV2TableError(operation: String): Throwable = {
-    new AnalysisException(s"$operation is only supported with v2 tables.")
+    new AnalysisException(s"$operation is only supported with v2 tables. To use" +
+      s" v2 tables, please config the catalog correctly using spark.sql.catalog" +
```

Comment:
if catalog is not configured correctly, I think users will see table not found exception?